internal/tui/user.go

```diff
@@ -0,0 +1,216 @@
+				m.step = stepLastName
+				m.focusCurrentInput()
+				return m, textinput.Blink
+			}
+		}
+		return m, cmd
+
+	case stepLastName:
+		var cmd tea.Cmd
+		m.lastName, cmd = m.lastName.Update(msg)
+		if key, ok := msg.(tea.KeyMsg); ok {
+			if key.Type == tea.KeyEnter {
+				m.answers["LastName"] = m.lastName.Value()
+				m.step = stepDisplayName
+				m.focusCurrentInput()
+				return m, textinput.Blink
+			} else if key.Type == tea.KeyLeft {
+				m.step = stepFirstName
+				m.firstName.Focus()
+				return m, textinput.Blink
+			}
+		}
+		return m, cmd
+
+	case stepDisplayName:
+		var cmd tea.Cmd
+		m.displayName, cmd = m.displayName.Update(msg)
+		if key, ok := msg.(tea.KeyMsg); ok {
+			if key.Type == tea.KeyEnter {
+				m.answers["DisplayName"] = m.displayName.Value()
+				m.step = stepEmail
+				m.focusCurrentInput()
+				return m, textinput.Blink
+			} else if key.Type == tea.KeyLeft {
+				m.step = stepLastName
+				m.lastName.Focus()
+				return m, textinput.Blink
+			}
+		}
+		return m, cmd
+
+	case stepEmail:
+		var cmd tea.Cmd
+		m.email, cmd = m.email.Update(msg)
+		if key, ok := msg.(tea.KeyMsg); ok {
+			if key.Type == tea.KeyEnter {
+				m.answers["Email"] = m.email.Value()
+				m.step = stepPassword
+				m.focusCurrentInput()
+				return m, textinput.Blink
+			} else if key.Type == tea.KeyLeft {
+				m.step = stepDisplayName
+				m.displayName.Focus()
+				return m, textinput.Blink
+			}
+		}
+		return m, cmd
+
+	case stepPassword:
+		var cmd tea.Cmd
+		m.password, cmd = m.password.Update(msg)
+		if key, ok := msg.(tea.KeyMsg); ok {
+			if key.Type == tea.KeyEnter {
+				m.answers["Password"] = m.password.Value()
+				m.step = userSummary
+				return m, nil
+			} else if key.Type == tea.KeyLeft {
+				m.step = stepEmail
+				m.email.Focus()
+				return m, textinput.Blink
+			}
+		}
+		return m, cmd
+
+	case userSummary:
+		if key, ok := msg.(tea.KeyMsg); ok {
+			switch key.String() {
+			case "enter":
+				return m, tea.Quit
+			case "left":
+				m.step = stepPassword
+				m.focusCurrentInput()
+				return m, textinput.Blink
+			}
+		}
+	}
+	return m, nil
+}
+
+func (m *UserCreateModel) View() string {
+	var b strings.Builder
+
+	switch m.step {
+	case stepFirstName:
+		b.WriteString("Enter First Name:\n")
+		b.WriteString(m.firstName.View())
+
+	case stepLastName:
+		b.WriteString("Enter Last Name:\n")
+		b.WriteString(m.lastName.View())
+
+	case stepDisplayName:
+		b.WriteString("Enter Display Name:\n")
+		b.WriteString(m.displayName.View())
+
+	case stepEmail:
+		b.WriteString("Enter Email:\n")
+		b.WriteString(m.email.View())
+
+	case stepPassword:
+		b.WriteString("Enter Password:\n")
+		b.WriteString(m.password.View())
+
+	case userSummary:
+		b.WriteString("\nSummary:\n")
+		for _, key := range []string{"FirstName", "LastName", "DisplayName", "Email", "Password"} {
+			val := strings.TrimSpace(m.answers[key])
+			if val == "" {
+				val = "[missing]"
+			}
+			b.WriteString(fmt.Sprintf("• %s: %s\n", key, val))
```

Comment:
Don't display the password in plain-text on this review, mask it using asterisk ***, and we may want to ask the user to confirm the password during input